frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.tsx

```diff
@@ -130,18 +136,21 @@ const DataConsumptionCard: React.FC<DashboardItemProps> = ({
       <DashboardCardBody isLoading={!dataConsumptionQueryResult}>
         {chartData.length > 0 ? (
           <div>
+            <span className="text-secondary">
+              {CHART_LABELS[sortByKpi]} {suffixLabel}
+            </span>
             <Chart
               themeColor={ChartThemeColor.purple}
               domain={{ y: [0, maxVal] }}
               domainPadding={{ x: [15, 20], y: [10, 10] }}
-              padding={{ top: 20, bottom: 40, left: 40, right: 17 }}
+              padding={{ top: 20, bottom: 40, left: 50, right: 17 }}
               height={280}
             >
               <ChartAxis style={{ tickLabels: { padding: 5, fontSize: 10 } }} />
               <ChartAxis
                 dependentAxis
                 tickValues={yTickValues}
-                tickFormat={(t) => `${t}${maxUnit}`}
+                tickFormat={(t) => `${t}`}
```

Comment:
```suggestion
```
we don't need tickformat now since its just displaying values

Reply:
Acknowledged